### DISCUSSION TITLE
Glances as a widget for 'gethomepage / homepage', but dose it actually work?

### DISCUSSION THREAD

**Discussion by Arelius-D** [nicolargo/glances#2787] [Q&A]
Hey guys,

_(sorry if a bit off-topic or so..)_

I wonder if anyone around here has managed to configure the `services.yml` for [Homepage](https://github.com/gethomepage/homepage/blob/main/docs/widgets/services/glances.md) and got desired output from `Glances` in bedded to your own page?

**If so, how did you actually pull that off?**
![image](https://github.com/nicolargo/glances/assets/168124713/8f856cde-2870-49d6-8a3d-53f077a2cd13)

```
---
- Glansec(TEST):
  - CPU:
        widget:
            type: glances
            url: http://10.0.0.10:61208
            version: 4
            metric: cpu
            diskUnits: bytes
            refreshInterval: 5000
            pointsLimit: 15
  - Network Usage:
        widget:
            type: glances
            url: http://10.0.0.10:61208
            metric: network:eth1
  - info:
        widget:
            type: glances
            url: http://10.0.0.10:61208
```
![image](https://github.com/nicolargo/glances/assets/168124713/bb903f7c-1780-4b49-8898-b1450a0d8a36)

**Comment by RazCrimson** [comment]
Check this out: https://gethomepage.dev/v0.8.3/more/troubleshooting/#general-troubleshooting-tips

Not sure if this might help: https://github.com/nicolargo/glances/issues/2763

@spiralham Were you able to resolve the issue in the end? If so, any chance this might be the same problem?

**Reply by spiralham** [reply]
@RazCrimson Ahh I missed the part about trailing slashes in my widget urls from that troubleshooting page - that was it!  Thanks, that would have taken me much longer to figure out.
